ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -127,8 +132,7 @@ export const itwEidIssuanceMachine = setup({
         onError: [
           {
             guard: "isSessionExpired",
-            actions: "handleSessionExpired",
-            target: "#itwEidIssuanceMachine.TosAcceptance"
+            target: "SessionExpired"
```

Comment:
I don't think this is correct: we recently removed the `SessionExpired` state. Maybe a merge error?

Reply:
We moved the checks from the `SessionExpired` directly to the state transitions here: https://github.com/pagopa/io-app/pull/6371